.github/workflows/autoreview.yaml

```diff
@@ -10,6 +10,10 @@ on:
       - main
       - master
 
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
```

Comment:
in Box I'm using `group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}`, not too sure what's the diff tbh but LGTM

Reply:
IMHO it's the same, as the [GitHub Contexts documentation](https://docs.github.com/en/actions/learn-github-actions/contexts) describes `github.head_ref`:
> The head_ref or source branch of the pull request in a workflow run. This property is only available when the event that triggers a workflow run is either pull_request or pull_request_target.